type-definitions/Immutable.d.ts

```diff
@@ -2440,7 +2440,7 @@ declare module Immutable {
 
     // Reading values
 
-    has(key: string): key is keyof TProps;
+    has(key: string | number | Symbol): key is keyof TProps;
```

Comment:
Shouldn't it be `symbol` (lowercase) as type, not as constructor (interface)?

Reply:
So is this what is holding up the merge?

Reply:
I think maybe `key is keyof TProps & string;` is best here. `Extract` was introduced in TS 2.8, so using it would break anyone using 2.7 or earlier.